src/smartWallet/impl/gnosisSafe/index.ts

```diff
@@ -24,24 +37,19 @@ export * from './param';
  */
 export class GnosisSafeWallet implements ISmartWallet {
   private readonly ownerWallet: IWallet | undefined;
-  private readonly defaultAddress: string | undefined;
   private readonly perChainRpcUrls: Map<string, string>;
-  private readonly perChainAddresses: Map<string, string>;
   private readonly perChainSafeInit: Map<string, ExclusiveInit<SafeBundle>>;
   private readonly perChainOwnersInit: Map<string, ExclusiveInit<ReadonlySet<string>>>;
+  private readonly safeSDKIsInit: Map<string, boolean | undefined>;
```

Comment:
```suggestion
  private readonly safeSdkIsInit: Map<string, boolean | undefined>;
```

note `Rpc`, `Url` above

Reply:
Ok